### ISSUE TITLE
Blind balance isn't in sync

### ISSUE THREAD

**Issue by kangoala** [particl/particl-desktop#1617]
In overview the balance of blind (private) shows way to many coins in it. When sending or convert coins the correct amount seems to be displayed. Also I can't get rid of the -1 locked coins. Overall the amount of coins as shown in the overview window seems not fitting together.
![bandicam 2019-07-28 08-38-06-809](https://user-images.githubusercontent.com/45103026/62003208-0dc53100-b114-11e9-9d9f-5d9f3ac15cad.jpg)
![bandicam 2019-07-28 08-37-52-425](https://user-images.githubusercontent.com/45103026/62003209-10278b00-b114-11e9-8c4e-b41e8b75a9f6.jpg)

**Comment by HenkSwardt** [comment]
Good day @kangoala 

Thank you for this submission, we had a look at the totals being shown and it will be resolved in the upcoming build 

Regards

**Comment by zaSmilingIdiot** [comment]
Using the v2.0.0-alpha2-testnet, is this still an issue?

**Comment by kangoala** [comment]
I deinstalled alpha1-testnet, deletet folder particl-market and installed alpha2-testnet.
The latest alpha2-testnet then started right away with the three wallets (default, market, mymarket) created during alpha1-testnet and still had a discrepancy in the coinbalance. I tried to empty all coins but didn't achieved it, because in the public address there were exactly 1 part left which I couldn't withdraw because there were insufficent balance. I guess this must have been somehow been related to the -1 balance shown in locked status before.
I then deleted (renamed) all previous wallets and reinstalled alpha2-testnet and started now with an empty / fresh wallet. So far I experienced no problems with the coin balance. However, for an unexperienced user the balances are quite confusing. For example I converted with one transfer parts to my anon address (e.g. 30 parts) and tried to pay three smaller items (e.g. 0.1 part). I only were able to buy the first item since then all my 30 parts got locked (waiting for confirmations). Would it be possible to show anything like "coin controle" for the anon balance (e.g. how many anon coins are on how many addresses)?
